[2024-04-09T05:38:07.043+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: liver_disease_pipeline_2.validate_data scheduled__2024-04-08T00:00:00+00:00 [queued]>
[2024-04-09T05:38:07.053+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: liver_disease_pipeline_2.validate_data scheduled__2024-04-08T00:00:00+00:00 [queued]>
[2024-04-09T05:38:07.053+0000] {taskinstance.py:2170} INFO - Starting attempt 1 of 2
[2024-04-09T05:38:07.068+0000] {taskinstance.py:2191} INFO - Executing <Task(PythonOperator): validate_data> on 2024-04-08 00:00:00+00:00
[2024-04-09T05:38:07.075+0000] {standard_task_runner.py:60} INFO - Started process 15164 to run task
[2024-04-09T05:38:07.077+0000] {standard_task_runner.py:87} INFO - Running: ['***', 'tasks', 'run', 'liver_disease_pipeline_2', 'validate_data', 'scheduled__2024-04-08T00:00:00+00:00', '--job-id', '238', '--raw', '--subdir', 'DAGS_FOLDER/pipeline.py', '--cfg-path', '/tmp/tmp56rs0nl2']
[2024-04-09T05:38:07.080+0000] {standard_task_runner.py:88} INFO - Job 238: Subtask validate_data
[2024-04-09T05:38:07.123+0000] {task_command.py:423} INFO - Running <TaskInstance: liver_disease_pipeline_2.validate_data scheduled__2024-04-08T00:00:00+00:00 [running]> on host 60be0e44a1a9
[2024-04-09T05:38:07.206+0000] {taskinstance.py:2480} INFO - Exporting env vars: AIRFLOW_CTX_DAG_EMAIL='admin@localhost.com' AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='liver_disease_pipeline_2' AIRFLOW_CTX_TASK_ID='validate_data' AIRFLOW_CTX_EXECUTION_DATE='2024-04-08T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-04-08T00:00:00+00:00'
[2024-04-09T05:38:07.254+0000] {logging_mixin.py:188} INFO - Moved Liver Patient Dataset (LPD)_train.csv to artifacts/validatedData
[2024-04-09T05:38:07.255+0000] {python.py:201} INFO - Done. Returned value was: 1
[2024-04-09T05:38:07.284+0000] {taskinstance.py:1138} INFO - Marking task as SUCCESS. dag_id=liver_disease_pipeline_2, task_id=validate_data, execution_date=20240408T000000, start_date=20240409T053807, end_date=20240409T053807
[2024-04-09T05:38:07.305+0000] {local_task_job_runner.py:234} INFO - Task exited with return code 0
[2024-04-09T05:38:07.336+0000] {taskinstance.py:3280} INFO - 1 downstream tasks scheduled from follow-on schedule check
